app/controllers/api/teams.php

```diff
@@ -423,8 +423,9 @@
 
         $response->setStatusCode(Response::STATUS_CODE_CREATED);
         $response->dynamic($membership
-            ->setAttribute('email', $email)
-            ->setAttribute('name', $name)
+            ->setAttribute('teamName', $team->getAttribute('name'))
+            ->setAttribute('userName', $user->getAttribute('name'))
+            ->setAttribute('userEmail', $user->getAttribute('email'))
```

Comment:
Previous:
```
->setAttribute('email', $email)
->setAttribute('name', $name)
```
            
Again just making sure, was this on purpose - not using variable?

Reply:
Yeah it was, there is so much happening before I would use `$name` again, that I wanted to be a 100% sure and use the object from the database 👍🏻

Reply:
Response